mad-dog/src/helpers.js

```diff
@@ -67,7 +70,14 @@ const addAndUpgradeUsers = async (
         const newUserId = await addUser(libs, userMetadata, USER_PIC_PATH)
         addedUserIds.push(newUserId)
         userId = newUserId
-        logger.info(`Found existing user: ${userId}`)
+        logger.info(`Created new user: ${userId}`)
+
+        // Poll dp to get all proper and expected user metadata as hardcoded metadata
+        // does not contain all necessary fields (blocknumber, track_blocknumber, ...)
+        const userWalletAddress = getLibsWalletAddress(libs)
+        await waitForIndexing()
```

Comment:
does this actually poll? this looks more like a delay where it calls waitForIndexing, pauses 5000 ms and then calls getUserAccount

Reply:
ah, yes you are right. this just waits 5s for indexing to occur. updating comment